tests/performance-test/grafana/qdr-servicemonitor.yml

```diff
@@ -11,3 +11,18 @@ spec:
   endpoints:
   - port: port-8672
     interval: 10s 
+
+---
```

Comment:
I don't like that we are adding a servicemonitor for each QDR service; ideally there should be a label that can identify all of them. Unfortunately I checked and there is not; not is is immediately obvious to me how we'd get a label added to those services. This will have to do for now, thanks!